OpenDreamRuntime/Procs/DreamProcArguments.cs

```diff
@@ -34,5 +36,35 @@ namespace OpenDreamRuntime.Procs {
             }
             return DreamValue.Null;
         }
+
+        public bool TryGetNamedArgument(string argumentName, [NotNullWhen(true)] out DreamValue? value) {
```

Comment:
You never actually set this to `null` (`DreamValue.Null` is still a value).
```suggestion
        public bool TryGetNamedArgument(string argumentName, out DreamValue value) {
```